common/flatpak-table-printer.c

```diff
@@ -226,10 +226,17 @@ flatpak_table_printer_print (FlatpakTablePrinter *printer)
       for (j = 0; j < row->len; j++)
         {
           Cell *cell = g_ptr_array_index (row, j);
-          if (cell->align < 0)
-            g_print ("%s%-*s", (j == 0) ? "" : " ", widths[j], cell->text);
+          if (flatpak_fancy_output ())
+            {
+            if (cell->align < 0)
```

Comment:
This and all the lines added below it also need indentation fixes to match the convention in the rest of the codebase.

Reply:
I’ve actually never seen this particular code style before so I got a bit confused. (No sleep probably had something to do with it too.) I think I got it right this time.